src/main/java/seedu/address/logic/parser/CliSyntax.java

```diff
@@ -6,10 +6,10 @@
 public class CliSyntax {
 
     /* Prefix definitions */
-    public static final Prefix PREFIX_NAME = new Prefix("n/");
-    public static final Prefix PREFIX_PHONE = new Prefix("p/");
-    public static final Prefix PREFIX_EMAIL = new Prefix("e/");
-    public static final Prefix PREFIX_ADDRESS = new Prefix("a/");
-    public static final Prefix PREFIX_TAG = new Prefix("t/");
+    public static final Prefix PREFIX_NAME = new Prefix("n: ");
```

Comment:
change prefix definition to "n:" instead of "n: " and follow suit for the rest of the prefixes

Reply:
Did it for `address` and `tag` prefixes;  it works.
Will update the other prefixes for now for the sake of consistency.

However, having a space between a prefix and its argument(s) is arguably still more natural. Therefore, it would be good to visit this enhancement.

Reply:
Updated prefixes for `name`, `phone` and `email`.